indico/modules/rb/__init__.py

```diff
@@ -80,7 +80,7 @@ def _topmenu_items(sender, **kwargs):
 @signals.menu.items.connect_via('event-management-sidemenu')
 def _sidemenu_items(sender, event, **kwargs):
     if config.ENABLE_ROOMBOOKING and event.can_manage(session.user):
-        yield SideMenuItem('room_booking', _('Room booking'), url_for('rb.event_booking_list', event), 50,
+        yield SideMenuItem('room_booking', _('Room bookings'), url_for('rb.event_booking_list', event), 50,
```

Comment:
not sure i like this change. IMHO this is simple about the process of booking rooms. so the title there does not imply single/multiple bookings..

Reply:
My idea was that this view will evolve in a place for managing room bookings associated to the event. When implementing the feature request in #6046, I will most likely suggest adding a way to link existing bookings to the event/sessions. I can further imagine this view to also allow unlinking bookings. Maybe is it premature to change menu entry name?